### ISSUE TITLE
Auto play and indicator not working 

### ISSUE THREAD

**Issue by HuiPing0521** [Mudassirraza912/react-native-image-slider-banner#8]
![image](https://user-images.githubusercontent.com/76928791/209424223-ef0a7ff8-82e2-4b67-9baa-dc5892615e41.png)
  <ImageSlider
        data={[
          {
            img: "https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQ5a5uCP-n4teeW2SApcIqUrcQApev8ZVCJkA&usqp=CAU",
          },
          {
            img: "https://thumbs.dreamstime.com/b/environment-earth-day-hands-trees-growing-seedlings-bokeh-green-background-female-hand-holding-tree-nature-field-gra-130247647.jpg",
          },
          {
            img: "https://cdn.pixabay.com/photo/2015/04/19/08/32/marguerite-729510__340.jpg",
          },
        ]}
        autoPlay={true}
        onItemChanged={(item) => console.log("item", item)}
        closeIconColor="#fff"
      />

**Comment by HuiPing0521** [comment]
@Mudassirraza912 Hi ! Can you please check on it ? is it not working on react native web?

**Comment by Mudassirraza912** [comment]
@HuiPing0521 Yes this plugin has not been designed to be used with react-native-web.

**Comment by FurqanHameed399** [comment]
carousel is working fine on auto scroll but when user manually scroll slider it not working well as it stops slider in middle. like half slider 2 and half slider 3

![image](https://github.com/Mudassirraza912/react-native-image-slider-banner/assets/147412312/50786500-7dbc-4547-95c4-9858103224ac)

**Comment by eSparkMRaza** [comment]
@FurqanHameed399 can you show your code how you implement this because in my case its working fine. you can check [here](https://snack.expo.dev/@mraza912/bf2b39?platform=android).
